epochStart/bootstrap/shardStorageHandler.go

```diff
@@ -261,13 +251,13 @@ func (ssh *shardStorageHandler) saveLastCrossNotarizedHeaders(meta *block.MetaBl
 	}
 
 	crossNotarizedHdrs := make([]bootstrapStorage.BootstrapHeaderInfo, 0)
-	crossNotarizedHdrs = append(crossNotarizedHdrs, bootstrapStorage.BootstrapHeaderInfo{
-		ShardId: core.MetachainShardId,
-		Nonce:   neededHdr.GetNonce(),
-		Hash:    shardData.LastFinishedMetaBlock,
-	})
-
 	if len(shardData.PendingMiniBlockHeaders) > 0 {
+		crossNotarizedHdrs = append(crossNotarizedHdrs, bootstrapStorage.BootstrapHeaderInfo{
```

Comment:
I think that also lines between 238-251 could be moved under this if, but anyway they are completely harmless even if they would stay for now there.

Reply:
done